livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/llm.py

```diff
@@ -481,6 +484,42 @@ def with_telnyx(
             tool_choice=tool_choice,
         )
 
+    @staticmethod
+    def with_meta(
+        *,
+        model: str | MetaChatModels = "Llama-4-Maverick-17B-128E-Instruct-FP8",
+        api_key: str | None = None,
+        base_url: str = "https://api.llama.com/compat/v1/",
+        client: openai.AsyncClient | None = None,
+        user: NotGivenOr[str] = NOT_GIVEN,
+        temperature: NotGivenOr[float] = NOT_GIVEN,
+        parallel_tool_calls: NotGivenOr[bool] = NOT_GIVEN,
+        tool_choice: ToolChoice = "auto",
```

Comment:
I think we should default this to:
`NotGivenOr[ToolChoice] = NOT_GIVEN,`

Reply:
So I don't think so. Llama will throw an error if tool_choice is not set to `auto`

![image](https://github.com/user-attachments/assets/0fe43221-1669-4861-bfb2-91058594fac1)

However that also means we can get rid of this arg.